pages/index.js

```diff
@@ -6,6 +6,7 @@ import Link from "next/link";
 import ServerError from "../components/errors/serverError";
 import Avatar from "../assets/images/user.png";
 import config from "../config";
+import moment from "moment";
```

Comment:
Can you use another way for setting date formats, as we are not using moment library anymore.